site/content/docs/developing/environment-variables.ja.md

```diff
@@ -58,18 +58,39 @@ environments:
 
 さらに、環境変数をまとめて追加したい場合、 [env file](https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) に環境変数を追加します。そして、ファイルパス (Workspace の root からのファイルパス) を [Manifest](../manifest/overview.ja.md) の `env_file` フィールドに記述します。
 
+メインコンテナのワークスペースのルートや、任意のサイドカーコンテナ定義、`logging` フィールド下に、env file を指定して、Firelens サイドカーコンテナに env file を設定できます。
```

Comment:
I prefer "env_file を指定して" because the field name is `env_file` which includes the underbar.
Sorry for nitpicking: Firelens to FireLens
And finally I prefer "サイドカーコンテナに環境変数ファイルを設定できます。"

Reply:
Understood,  I reflect your feedback.